src/pages/tasks/NewTaskDescriptionPage.js

```diff
@@ -91,7 +94,7 @@ function NewTaskDescriptionPage(props) {
                 >
                     <View style={styles.mb5}>
                         <TextInput
-                            defaultValue={props.task.description}
+                            defaultValue={parser.htmlToMarkdown(parser.replace(props.task.description))}
```

Comment:
Huh? Why are we converting to HTML and back to markdown?? This was not part of the proposal, was it?

Reply:
I have mentioned in this https://github.com/Expensify/App/pull/29144#issuecomment-1754417578

Reply:
- Yeah. Comments and Privates notes are saved on BE side as HTML. As I mentioned above, with the description, I found out that if we let BE store the HTML, for example,
<blockscope>Hello <br/> 123</blockscope>, it just stores as "Hello123"
- So if we want to store description as HTML, we need the BE fix

Reply:
@iwiznia just requested confirm here https://github.com/Expensify/App/issues/28342